### PR TITLE
Subnet Editing (tagging) #1283

### PR DESCRIPTION
**Pull request by ATGE**

### CODE REVIEW

File: SoftLayer/managers/network.py [hunk 1]
```diff
@@ -226,6 +237,20 @@ def cancel_subnet(self, subnet_id):
         billing_id = subnet['billingItem']['id']
         return self.client['Billing_Item'].cancelService(id=billing_id)
 
+    def set_tags_subnet(self, subnet_id, tags):
+        """Tag a subnet by passing in one or more tags separated by a comma.
+
+        :param int subnet_id: The ID of the subnet.
```
Comment (allmightyspiff) [inline SoftLayer/managers/network.py:243]: Add the `tags` param documentation

File: SoftLayer/managers/network.py [hunk 2]
```diff
@@ -226,6 +237,20 @@ def cancel_subnet(self, subnet_id):
...
+        """
+        return self.subnet.setTags(tags, id=subnet_id)
+
+    def edit_note_subnet(self, subnet_id, note):
+        """Edit the note for this subnet.
+
+        :param int subnet_id: The ID of the subnet.
```
Comment (allmightyspiff) [inline SoftLayer/managers/network.py:250]: Add the `tags` param documentation

File: SoftLayer/managers/network.py (hunk 1 above)
Comment (ATGE) [reply SoftLayer/managers/network.py:248]: done

File: SoftLayer/managers/network.py (hunk 2 above)
Comment (ATGE) [reply SoftLayer/managers/network.py:256]: done

**Pull request merged**